src/parallel.cpp

```diff
@@ -34,20 +35,14 @@ void AddScriptCheckThreads(int i, CCheckQueue<CScriptCheck>* pqueue)
 
 void AddAllScriptCheckQueuesAndThreads(int nScriptCheckThreads, boost::thread_group* threadGroup)
 {
-    vector<CCheckQueue<CScriptCheck>* > vScriptCheckQueue;
```

Comment:
this is not really an important change because this code is not performance related.  But the vector STL object is going to malloc and free each time you push_back (or over-malloc).  But since this use is a known constant size array, this really good time to use a traditional c array.  Personally, I always prefer to use the simplest tool that gets the job done, even though in this case it doesn't really matter from a performance perspective.  But, this change is not very important; take it or leave it, your preference.